R/spinMyR.R

```diff
@@ -213,8 +215,14 @@ spinMyR <- function(file, wd, outDir, figDir, outSuffix,
               fileMd,
               quiet = !verbose,
               envir = spinMyR_Env)
+  if (!keepRmd) {
+    unlink(fileRmd)
```

Comment:
perhaps `suppressWarnings(unlink(figDirName, recursive = TRUE))` is a bit more error-prone?